extensions/wikia/FandomComMigration/FandomComMigrationHooks.class.php

```diff
@@ -46,8 +46,15 @@ public static function onWikiaSkinTopScripts( &$vars, &$scripts ) {
 	}
 
 	private static function isEnabled() {
-		global $wgFandomComMigrationScheduled, $wgFandomComMigrationDone;
+		global $wgDomainChangeDate, $wgFandomComMigrationScheduled, $wgFandomComMigrationDone;
+
+		if ( $wgFandomComMigrationDone && !empty( $wgDomainChangeDate ) ) {
+			$migrationDateTime = new DateTime( $wgDomainChangeDate );
+			$weekAgo = ( new DateTime() )->sub( new DateInterval( 'P7D' ) );
+
+			return $migrationDateTime > $weekAgo;
+		}
 
-		return !empty( $wgFandomComMigrationScheduled ) || $wgFandomComMigrationDone;
+		return !empty( $wgFandomComMigrationScheduled );
```

Comment:
what if `wgFandomComMigrationDone` is set to true but the `wgDomainChangeDate` is empty?

Reply:
Then the data is corrupted and it's better to not show the banner than show it forever.